### ISSUE TITLE
Ghost CLI uses utf8mb4_general_ci as collation when creating MySQL8 database but documentation suggests utf8mb4_0900_ai_ci

### ISSUE THREAD

**Issue by cxcorp** [TryGhost/Ghost-CLI#1718]
### Summary

When creating a new MySQL 8 database, ghost-cli uses `utf8mb4_general_ci` as the collation.

https://github.com/TryGhost/Ghost-CLI/blob/2bb51b75b1f448ff99d6def016c7e76ed79a5c28/extensions/mysql/index.js#L250

The Ghost [How to update MySQL 5 to MySQL 8](https://ghost.org/docs/faq/supported-databases/#how-to-update-mysql-5-to-mysql-8) guide instructs users to migrate table collations to use `utf8mb4_0900_ai_ci` instead. The Ghost [MySQL troubleshooting FAQ](https://ghost.org/docs/faq/troubleshooting-mysql-database/#error-er_cant_create_table-with-_foreign-key-constraint-is-incorrectly-formed_) also suggests that `utf8mb4_general_ci` is the default for MySQL 5.7 and `utf8mb4_0900_ai_ci` for MySQL 8.

In summary, there is a disparity between what the CLI does and what the documentation suggests. I'm not sure which one is wrong.

### Technical details

This is automatically output by Ghost-CLI if an error occurs, please copy & paste:

* OS: n/a
* Node Version: n/a
* Ghost-CLI Version: 2bb51b75b1f448ff99d6def016c7e76ed79a5c28
* Environment: n/a
* Command: n/a

## Bug submission checklist

Please fill out this checklist to acknowledge that you followed the requirements to submit a bug report.

- [X] Tried to find help in the forum & docs
- [X] Checked for existing issues
- [ ] Attached log file
- [X] Provided technical details incl. operating system

**Comment by ErisDS** [comment]
Hey @cxcorp, thanks for raising this, it seems this is likely to be the root cause of a lot of issues.

A PR to fix this would be very welcome.
